lib/models/inquiry_request_body.dart

```diff
@@ -0,0 +1,15 @@
+import 'package:freezed_annotation/freezed_annotation.dart';
+
+part 'inquiry_request_body.freezed.dart';
+part 'inquiry_request_body.g.dart';
+
+@freezed
+class InquiryRequestBody with _$InquiryRequestBody {
+  const factory InquiryRequestBody({
+    required String title,
+    required String content,
+  }) = _InquiryRequestBody;
+
+  factory InquiryRequestBody.fromJson(Map<String, dynamic> json) =>
+      _$InquiryRequestBodyFromJson(json);
```

Comment:
Retrofit의 한계로 `toJson` 메소드가 명시되어 있지 않으면 `@Body()`가 제대로 동작하지 않습니다... 단순히 오버라이드 해주기만 해도 인식할 수 있으니 추가만 해주면 됩니다. 이게 해결되면 아마 임시로 설정하신 `content-type` 헤더를 없애도 동작할 거에요.

참고로 `content-type` 헤더는 일반적인 케이스에 대해서는 dio 내부에서 자동으로 결정되는 것으로 알고 있습니다.

```suggestion
  factory InquiryRequestBody.fromJson(Map<String, dynamic> json) =>
      _$InquiryRequestBodyFromJson(json);

  @override
  Map<String, dynamic> toJson();
```

Reply:
아하 이게 문제였군요... 안 그래도 다른 모델 중에서도 toJson을 오버라이딩 하신게 의문이었는데 같은 이유겠군요 ㅇㅁ ㅇ